lemminx-maven/src/test/java/org/eclipse/lemminx/maven/test/LocalPluginTest.java

```diff
@@ -108,7 +108,7 @@ public void testCompleteConfigurationParametersDuplicates() throws IOException,
 		List<CompletionItem> completions = languageService.doComplete(createDOMDocument("/pom-duplicate-configuration-completion.xml", languageService),
 				new Position(15, 4), new SharedSettings())
 			.getItems();
-		assertTrue(completions.stream().map(CompletionItem::getLabel).filter("compilePath"::equals).count() == 1);
+		assertTrue(completions.stream().map(CompletionItem::getLabel).filter("annotationProcessors"::equals).count() == 1);
```

Comment:
compilePath is not actually an editable parameter for maven-compiler-plugin, so this test was faulty